packages/pilot/src/components/MetricChart/sizePropValidation.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line consistent-return
+const isPercentageOrNumber = (props, propName, componentName) => {
+  const prop = props[propName]
+
+  const isString = typeof prop === 'string'
+  const isNumber = typeof prop === 'number'
+
+  const validationError = message => new Error(
+    `Invalid prop '${propName}' supplied to '${componentName}'. ${message}`
+  )
+
+  if (!isString && !isNumber && propName !== 'width') {
+    return validationError('Expected a percentage string or number.')
+  }
+
+  const isPercentage = /\d%/
+  if (isString && !isPercentage.test(prop)) {
+    return validationError('Expected string to be a percentage value.')
+  }
```

Comment:
Aqui seria melhor só colocar um `return` no final da função para evitar o `eslint-disable` no começo da func.
```suggestion
  }
  return undefined
```

Reply:
Feito!